pkg/skaffold/deploy/helm.go

```diff
@@ -57,6 +59,10 @@ func NewHelmDeployer(runCtx *runcontext.RunContext) *HelmDeployer {
 	return &HelmDeployer{
 		HelmDeploy:  runCtx.Cfg.Deploy.HelmDeploy,
 		kubeContext: runCtx.KubeContext,
+		kubectl: kubectl.CLI{
```

Comment:
I'm a little confused by the alias you have above. you do 

```golang
kubectl    kubectl.CLI
```
but then here you have
```golang
kubectl: kubectl.CLI{
```
so is it actually `kubectl.CLI.CLI`? maybe we can just get rid of the alias?